tests/Schema/SchemaDiffTest.php

```diff
@@ -1,134 +0,0 @@
-<?php
-
-namespace Doctrine\DBAL\Tests\Schema;
-
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Schema\ForeignKeyConstraint;
-use Doctrine\DBAL\Schema\SchemaDiff;
-use Doctrine\DBAL\Schema\Sequence;
-use Doctrine\DBAL\Schema\Table;
-use Doctrine\DBAL\Schema\TableDiff;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-
-class SchemaDiffTest extends TestCase
```

Comment:
Why are these tests removed?

Reply:
These tests mock the methods that are no longer used for schema generation. The tests that make assertions against SQL provide more harm than good, so it is not worth updating them according to the code changes.